src/main/java/org/geysermc/platform/fabric/item/FabricItemRegistry.java

```diff
@@ -0,0 +1,32 @@
+package org.geysermc.platform.fabric.item;
+
+import org.geysermc.connector.network.translators.item.ItemEntry;
+import org.geysermc.connector.network.translators.item.ItemRegistry;
+
+import com.nukkitx.protocol.bedrock.packet.StartGamePacket;
+
+import io.github.theepicblock.polymc.api.register.PolyRegistry;
+import net.minecraft.item.BlockItem;
+import net.minecraft.item.Item;
+import net.minecraft.util.registry.Registry;
+
+public class FabricItemRegistry {
+	
+	public static void registerItems() {
+		for (int count = 1; count <= Registry.ITEM.getEntries().size(); count++) {
+			Item currentItem = Registry.ITEM.get(count);
+			if (!Registry.ITEM.getId(currentItem).getNamespace().equals("minecraft")) {
+				if (!(currentItem instanceof BlockItem)) {
+					ItemRegistry.ITEMS.add(new StartGamePacket.ItemEntry(Registry.ITEM.getId(currentItem).getNamespace() + ":" + Registry.ITEM.getId(currentItem).getPath(), (short) ((short)Registry.ITEM.getRawId(currentItem))));
+					ItemRegistry.ITEM_ENTRIES.put(count, new ItemEntry(Registry.ITEM.getId(currentItem).getNamespace() + ":" + Registry.ITEM.getId(currentItem).getPath(), Registry.ITEM.getRawId(currentItem), Registry.ITEM.getRawId(currentItem), 0, false));

```

Comment:
You can consolidate the if/else and just set the `isBlock` value to `currentItem instanceof blockItem`.